cephfs/admin/response.go

```diff
@@ -55,6 +73,11 @@ func (r response) Status() string {
 // that response is no longer needed for processing.
 func (r response) End() error {
 	if !r.Ok() {
+		if ce, ok := r.err.(cephError); ok {
```

Comment:
is it not recommended to use `errors.Is()` and `errors.As()`?

Reply:
I think that would only be needed if I wanted to check if r.err OR any of the "children" returned by an Unwrap call matched cephError. In this case I know I only want to check `r.err` itself, and this is all fairly internal. Plus, it means that the library could still also run on older versions of Go that don't have errors.As. (But not the tests because I used errors.As there... but those are tests we have more control over! :-))